pkg/app/pagewriter/error_page_test.go

```diff
@@ -28,41 +29,75 @@ var _ = Describe("Error Page Writer", func() {
 	Context("WriteErrorPage", func() {
 		It("Writes the template to the response writer", func() {
 			recorder := httptest.NewRecorder()
-			errorPage.WriteErrorPage(recorder, 403, "/redirect", "Access Denied")
+			errorPage.WriteErrorPage(recorder, ErrorPageOpts{
+				Status:      403,
+				RedirectURL: "/redirect",
+				RequestID:   requestID,
+				AppError:    "Access Denied",
+			})
 
 			body, err := ioutil.ReadAll(recorder.Result().Body)
 			Expect(err).ToNot(HaveOccurred())
-			Expect(string(body)).To(Equal("Forbidden You do not have permission to access this resource. /prefix/ 403 /redirect Custom Footer Text v0.0.0-test"))
+			Expect(string(body)).To(Equal("Forbidden You do not have permission to access this resource. /prefix/ 403 /redirect 11111111-2222-4333-8444-555555555555 Custom Footer Text v0.0.0-test"))
 		})
 
 		It("With a different code, uses the stock message for the correct code", func() {
 			recorder := httptest.NewRecorder()
-			errorPage.WriteErrorPage(recorder, 500, "/redirect", "Access Denied")
+			errorPage.WriteErrorPage(recorder, ErrorPageOpts{
+				Status:      500,
+				RedirectURL: "/redirect",
+				RequestID:   requestID,
+				AppError:    "Access Denied",
+			})
 
 			body, err := ioutil.ReadAll(recorder.Result().Body)
 			Expect(err).ToNot(HaveOccurred())
-			Expect(string(body)).To(Equal("Internal Server Error Oops! Something went wrong. For more information contact your server administrator. /prefix/ 500 /redirect Custom Footer Text v0.0.0-test"))
+			Expect(string(body)).To(Equal("Internal Server Error Oops! Something went wrong. For more information contact your server administrator. /prefix/ 500 /redirect 11111111-2222-4333-8444-555555555555 Custom Footer Text v0.0.0-test"))
 		})
 
 		It("With a message override, uses the message", func() {
 			recorder := httptest.NewRecorder()
-			errorPage.WriteErrorPage(recorder, 403, "/redirect", "Access Denied", "An extra message: %s", "with more context.")
+			errorPage.WriteErrorPage(recorder, ErrorPageOpts{
+				Status:      403,
+				RedirectURL: "/redirect",
+				RequestID:   requestID,
+				AppError:    "Access Denied",
+				Messages: []interface{}{
+					"An extra message: %s",
+					"with more context.",
+				},
```

Comment:
This is less nice, though I don't have a better suggestion :thinking: Having to define the list like this does detract from the usefulness of the format stringing within the pagewriter. Perhaps we want to just have message as a string and not bother?

Reply:
Let me try to make this a string, while keeping the function that wrapped this in our actual codebase using a `...interface{}` -- I think explicit tests were the only spot this ugliness showed itself since we wrapped it in the other spots.

Reply:
It looks like this is the only spot that had warts -- leaving it as `[]interface{}` led to easy consumption of `messages ...interface{}` in all the other uses of this argument.

Should we make a helper method in this test to simulate that we are consuming a variable length argument in all uses of this field?